mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -196,14 +198,15 @@ void rock::buildBackendPipeline(OpPassManager &pm,
   floatEmuOpts.sourceTypeStrs = unsupportedFloats;
   floatEmuOpts.targetTypeStr = "f32";
   gpuPm.addPass(arith::createArithEmulateUnsupportedFloats(floatEmuOpts));
-  if (archInfo.hasFp8ConversionInstrs) {
-    ArithToAMDGPUConversionPassOptions options;
-    options.saturateFP8Truncf = true;
-    gpuPm.addPass(createArithToAMDGPUConversionPass(options));
+  ArithToAMDGPUConversionPassOptions arithOptions;
+  arithOptions.chipset = options.chip;
+  arithOptions.allowPackedF16Rtz = true;
+  arithOptions.saturateFP8Truncf = true;
+  gpuPm.addPass(createArithToAMDGPUConversionPass(arithOptions));
+  if (archInfo.hasFp8ConversionInstrs)
     gpuPm.addPass(createArithToAMDGPUConversionPass());
```

Comment:
Do we need this if statement anymore? We might want the else branch, but that's about it.